doc/conf.py

```diff
@@ -261,9 +261,9 @@
     "pygments_dark_style": "monokai",
     "logo": {
         "alt_text": "scikit-learn homepage",
-        "image_relative": "logos/scikit-learn-logo-small.png",
-        "image_light": "logos/scikit-learn-logo-small.png",
-        "image_dark": "logos/scikit-learn-logo-small.png",
+        "image_relative": "logos/scikit-learn-logo-without-subtitle.svg",
+        "image_light": "logos/scikit-learn-logo-without-subtitle.svg",
+        "image_dark": "logos/scikit-learn-logo-without-subtitle.svg",
```

Comment:
I see that we have an additional occurrence in this file (related to open graph extension). I think that we can change it as well and remove the PNG file then.

Reply:
If you mean line 886 `ogp_image = "https://scikit-learn.org/stable/_static/scikit-learn-logo-small.png"`, it seems like this is for thumbnails. I need to find out how to test it if I change it. 
Would it be better to address that on another PR @glemaitre?

Reply:
A thought: for social media previews and such, which is what L886 is I think, it might be useful to use a PNG and not a SVG? Just to increase the chances of the preview "just working"?